From 7b744c8a17467ba805fcad0ee6069ddab1d559b3 Mon Sep 17 00:00:00 2001
From: Shijith Thotton <sthotton@marvell.com>
Date: Thu, 8 Dec 2022 12:53:30 +0530
Subject: [PATCH 069/955] examples/l2fwd-event: fix worker cleanup function
 args

Fixed l2fwd worker cleanup function call arguments. Number of events
enqueued and dequeued are set the opposite.

Fixes: 622ebb6b4a7d ("examples/l2fwd-event: clean up worker state before exit")

Signed-off-by: Shijith Thotton <sthotton@marvell.com>
Change-Id: Ibe2a6ad0c9f84d33193332fa0a8014400b4001ab
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/dataplane/dpdk/+/92123
Tested-by: sa_ip-toolkits-Jenkins <sa_ip-toolkits-jenkins@marvell.com>
Reviewed-by: Pavan Nikhilesh Bhagavatula <pbhagavatula@marvell.com>
---
 examples/l2fwd-event/l2fwd_event.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/examples/l2fwd-event/l2fwd_event.c b/examples/l2fwd-event/l2fwd_event.c
index 63450537fe30a..4b5a032e3530d 100644
--- a/examples/l2fwd-event/l2fwd_event.c
+++ b/examples/l2fwd-event/l2fwd_event.c
@@ -284,7 +284,7 @@ l2fwd_event_loop_burst(struct l2fwd_resources *rsrc,
 		}
 	}
 
-	l2fwd_event_worker_cleanup(event_d_id, port_id, ev, nb_rx, nb_tx, 0);
+	l2fwd_event_worker_cleanup(event_d_id, port_id, ev, nb_tx, nb_rx, 0);
 }
 
 static __rte_always_inline void
@@ -468,7 +468,7 @@ l2fwd_event_loop_vector(struct l2fwd_resources *rsrc, const uint32_t flags)
 		}
 	}
 
-	l2fwd_event_worker_cleanup(event_d_id, port_id, ev, nb_rx, nb_tx, 1);
+	l2fwd_event_worker_cleanup(event_d_id, port_id, ev, nb_tx, nb_rx, 1);
 }
 
 static void __rte_noinline
-- 
2.25.1

